contracts/tokenbridge/ethereum/L1AtomicTokenBridgeCreator.sol

```diff
@@ -283,7 +312,6 @@ contract L1AtomicTokenBridgeCreator is Initializable, OwnableUpgradeable {
         uint256 maxGas,
         uint256 gasPriceBid
     ) internal returns (uint256) {
-        address l2ProxyAdminOwner = _getRollupOwner(inbox);
         L2CreationCode memory l2Code = L2CreationCode(
             _creationCodeFor(l2RouterTemplate.code),
             _creationCodeFor(l2StandardGatewayTemplate.code),
```

Comment:
it would be cheaper if you wrap these logic into the child contract (retryableSender) 
passing 5 huge contracts vs 5 address

this is cleaner tho

Reply:
Regarding `retryableSender`, right now it is being one-time deployed behind proxy in `initialize`, with msg.sender as owner. So it could be upgraded, even tho its implementation is made to be as simple as possible

Reply:
I see, then having these on the `retryableSender ` to save some L1 gas seems to make some sense.

Reply:
done, moved calculation of data, submission cost and value to retryableSender, along with refund logic